apps/sms/views/shared/js/activity_handler.js

```diff
@@ -366,7 +364,7 @@ var ActivityHandler = {
               break;
             }
 
-            return text || formatValue('mms-message');
```

Comment:
ah yeah, I see

but in this specific case we don't really care as we're sending a Notification :)